profiles/apparmor/template.go

```diff
@@ -44,7 +44,7 @@ profile {{.Name}} flags=(attach_disconnected,mediate_deleted) {
 
 {{if ge .Version 208095}}
   # suppress ptrace denials when using 'docker ps' or using 'ps' inside a container
-  ptrace (trace,read) peer={{.Name}},
+  ptrace (trace,read,tracedby,readby) peer={{.Name}},
```

Comment:
I think maybe the comment should be changed on the line above - these are I think now needed as we allow `ptrace`  which we did not for seccomp reasons before kernel 4.8, and I don't think you would get these just from `ps`.